lib/vcloud/core/vapp.rb

```diff
@@ -164,6 +164,16 @@ def power_on
         running?
       end
 
+      # Power off vApp
+      #
+      # @return [Boolean] Returns true if the VM is powered off, false otherwise
+      def power_off
+        raise "Cannot power off a missing vApp." unless id
```

Comment:
How about "vApp does not exist - skipping..." or something rather than "Cannot power off a missing vApp"? As it stands, seeing that repeated a few times over and over again might cause panic when reviewing build output.